### PR TITLE
Strip installations in Dockerfile

### PR DESCRIPTION
**Pull request by dominiklohmann**
### :notebook_with_decorative_cover: Description

This shaves an additional ~10 MB off the resulting Docker images' size.

```
$ docker images
```

|REPOSITORY|TAG|IMAGE ID|CREATED|SIZE|
|-|-|-|-|-|
|vast-topic/strip-install-docker|latest|d89c14eb2291|5 seconds ago|162MB|
|vast-master|latest|40057e003de4|6 minutes ago|171MB

### :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

Decide whether or not we want this.

### CODE REVIEW

**Review by mavam — APPROVED** [review]
The change looks reasonable, but can you elaborate on the downsides, e.g., visibility during debugging?

**Comment by dominiklohmann** [comment]
> The change looks reasonable, but can you elaborate on the downsides, e.g., visibility during debugging?

First step to debug would likely be to reproduce outside of a Docker container, which is why I think this does no harm. I'll wait for @lava or @tobim to give their approval, though.

**Pull request merged**
